client/src/Day.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import $ from 'jquery';
+class Day extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {toggle: 0, from: ''};
+
+    this.sendDayFrom = this.sendDayFrom.bind(this);
+  }
+
+
+  sendDayFrom() {
+
+    var options = { month: 'numeric', day: 'numeric', year: 'numeric' };
+    var r = new Date(this.props.data.date).toLocaleDateString(undefined, options);
+    console.log(r);
+    this.props.changeDate(r);
+
+
+
+
+    //toggle if once was clicked set second date
+  }
+
+  render() {
+    var c;
```

Comment:
If you are not using this variable, you could delete it ;-)

Reply:
Done it!